posts/2022-06-07-real-time-data-sharing-pros-cons.md

```diff
@@ -0,0 +1,131 @@
+
+
+
+3. **Better partnerships**
+
+Partners often rely on having a trusted, single source of the truth that they align around as they work together. Those that do this well are successful, profitable, and stay together. By integrating deeper with your partners’ data around a shared truth, you can start thinking through the journey holistically instead of in bite sizes. This leads to a better understanding of your partners and their challenges, it removes distrust in the business relationship, and ensures everyone is aware of their responsibilities and maximizes their profit. 
+
+
+## The cons of real-time data sharing
+
+**Anyone who works with data knows that volume and speed requirements play a big part in how time-consuming, difficult, and expensive it is to build and maintain the infrastructure supporting it. Here are some of the cons of real-time data sharing:**
+
+
+
+1. **Time consuming and complex to build**
+
+    True real-time data sharing can be extremely hard to design, build, and implement across multiple partners. While point-to-point APIs between two partners is somewhat straightforward, but when multiple parties get involved, the challenges grow exponentially, which makes it time-consuming and costly to build. 
+
+
+2. **Difficult to create a shared and trusted source of truth**
+
+    Creating a single source of truth that all parties trust is no straight-forward task. In order for this to happen, all partners will need the ability to view and audit all data without the ability to alter the data itself. This demands a highly complex solution that not only reconciles all incoming data in real time to determine what’s truly the latest, it also requires strict data governance and logging of every transaction and change ever taken place.   
+
+
+3. **Expensive to build and maintain**
+
+    In order for real-time data to work, your system needs to have high availability of at least 99.99% of the time to ensure all data is truly up to date at all times. Building a system that is redundant, resilient, secure, and scalable, can be extremely costly and time-consuming to build and maintain in-house.
+
+
+
+## How Vendia streamlines real-time data sharing
+
+Vendia has built a B2B data-sharing platform that gives you all the benefits of real-time data sharing without all the headaches. Vendia has designed a platform that brings all data sharing parties together around a single source of truth that is updated in real-time.
+
+**Easy integration through a single access point**
+
+Every partner gets an access point (a node) to the Vendia Share platform, with access to their own copy of the latest data across the data-sharing partnership. Every partner can easily connect their system (whether cloud or on-premise) to Vendia, no need to build or maintain multiple API connection points, and you can be up and running in a matter of days. 
```

Comment:
`on-premise` is mostly aspirational, at least compared with the quality of our cloud integrations